setup.py

```diff
@@ -25,6 +26,15 @@ def grep(attrname):
 
 file_text = read(fpath('arrow/__init__.py'))
 
+install_requires = ['backports.functools_lru_cache',
```

Comment:
`backports.functools_lru_cache` is only required for Python 2.x.  Could you do a similar `install_requires.append` as you do for chai?